starter-code/routes/movies.js

```diff
@@ -0,0 +1,88 @@
+const express = require('express');
+const router  = express.Router();
+const Movie = require('../models/movie');
+
+router.get('/movies', (req, res, next) => {
+	Movie.find()
+	.then(movies => {
+		console.log(movies)
+		res.render('movies/index', {movies});
+	})
+	.catch(error => {
+		res.render(error);
+	})
+});
+
+router.get('/movies/:id', (req, res, next) => {
+	let movieId = req.params.id;
+	Movie.findOne({'_id': movieId})
+	.then(movie => {
+		console.log(movie)
+		res.render('movies/show', {movie});
+	})
+	.catch(error => {
+		res.render('error');
+	})
+});
+
+
+router.get('/movie/new', (req, res, next) => {
+	Movie.find()
+	.then(movies => {
+		console.log(movies)
+		res.render('movies/new', {movies});
+	})
+	.catch(error => {
+		res.render('error');
+	})
+});
+
+router.post('/movie/new', (req, res, next) => {
+	const { title, director, stars, genre, description, spoilers } = req.body;
+	const newCeleb = new Movie({ title, director, stars, genre, description, spoilers });
+	newCeleb.save()
+	.then((movie) => {
+		res.redirect('/movies');
+	})
+	.catch((error) => {
+		res.render('error');
```

Comment:
Aqui, é bom também enviar o erro para a página, não o erro completo, mas ao menos a mensagem que indica o que esteve errado, por exemplo "error saving movie on DB" e nao o típico "There was an error on the server! Check the console!"

.catch((error) => {
		res.render('error', { error });